decentraland/unity-explorer
# feat: added prefab and base view for layer panel section

## What does this PR change?

This PR adds a new panel in the bottom left of the map to control the layer visibility and map type.
So far only some layers have been integrated such as:
* minigames
* POIs
* favorites
* people


## How to test the changes?



1. Launch the explorer
2. Open the map
3. Verify that the new button in bottom left opens and closes the layer panel
4. Verify that the previously said layers are working when toggling them on and off

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
